.github/workflows/build.yml

```diff
@@ -21,6 +21,7 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version: 20
+          cache: 'npm'
```

Comment:
@Thilo-Acrolinx Cache is of no use. We generally use npm ci on CI pipelines, which deletes the node_modules dir
https://docs.npmjs.com/cli/v10/commands/npm-ci